commands-v2/ramsete/constants.py

```diff
@@ -0,0 +1,61 @@
+from commands2 import RamseteCommand
+
+from wpilib import ADXRS450_Gyro
+
+from wpimath.kinematics import DifferentialDriveKinematics
+
+from math import pi
```

Comment:
I would personally prefer it if it were spelt `math.pi` (to make it less of an inconvenience to use the math functions) but I don't really care that much.

Reply:
Ok, I can fix it.